.github/workflows/ci.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
       run: cargo test --no-run --locked
 
     - name: Test
-      run: cargo test
+      run: cargo test -- --nocapture
```

Comment:
Do we still need this?

Reply:
Yup! I believe tests shouldn't print anything by default anyway. I'd say it's even a cargo bug that `--nocapture` is a thing at all :-)